pkgs/by-name/in/intentrace/package.nix

```
@@ -0,0 +1,31 @@
+{
+  fetchFromGitHub,
+  lib,
+  rustPlatform,
+}:
+
+let
+  version = "0.2.4";
+in
+rustPlatform.buildRustPackage {
+  inherit version;
+  pname = "intentrace";
+
+  src = fetchFromGitHub {
+    owner = "sectordistrict";
+    repo = "intentrace";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-Bsis8tL2xahJT/qAFVbbd/CZ7n8KJYLPTIl1a1WHR4c=";
+  };
+
+  cargoHash = "sha256-pyGcQy7p0+Vqv3Khy1hLgahcOpqnbKKmRLZcKwkvVWw=";
+
+  meta = {
+    description = "Prettified Linux syscall tracing tool (like strace)";
+    homepage = "https://github.com/sectordistrict/intentrace";
+    license = lib.licenses.mit;
+    platforms = lib.platforms.linux;
```

```suggestion
    platforms = [ "x86_64-linux" ];
```
It seems upstream currently hardcodes register names
```
error[E0609]: no field `rax` on type `&user_regs_struct`
   --> src/main.rs:326:43
    |
326 |     syscall.errno = errno_check(registers.rax);
    |                                           ^^^ unknown field
    |
    = note: available fields are: `regs`, `sp`, `pc`, `pstate`
```

---

That was also mentioned in the PR description. Good spot, worth fixing the nix to match